src/Sql/Sql/Database/Cmdlet/GetAzureSqlDatabase.cs

```diff
@@ -14,11 +14,13 @@
 
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Commands.Sql.Database.Model;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 using System.Collections.Generic;
 using System.Management.Automation;
 
 namespace Microsoft.Azure.Commands.Sql.Database.Cmdlet
 {
+    [GenericBreakingChange("Database property 'BackupStorageRedundancy' has been changed to 'CurrentBackupStorageRedundancy' and 'RequestedBackupStorageRedundancy' in this version.")]
```

Comment:
Since the breaking change is parameter-level, I prefer to use `CmdletParameterBreakingChangeAttribute`, please refer to https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#cmdletparameterbreakingchangeattribute

Reply:
The breaking change message aims to inform customer the breaking change in advance, so we should tell customer when breaking change is upcoming with date or version. I speculate the version of Az.Sql is 3.0.0 when breaking change is available.

Reply:
The cmdlet parameter (-BackupStorageRedundancy) will not change, only the return values (BackupStorageRedundancy => CurrentBackupStorageRedundancy, RequestedBackupStorageRedundancy) will be different.  

Therefore, instead of `CmdletParameterBreakingChangeAttribute`, I have changed the warning attribute to be `CmdletOutputBreakingChange` to reflect the updates in the cmdlet output.